python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -103,7 +111,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
we should have a tool to strip metadata out so things stay consistent. I'll write that this afternoon.